webapp/lib/view.dart

```diff
@@ -323,6 +335,11 @@ class ContentView {
       ..onClick.listen((_) => controller.command(controller.UIAction.tabSwitched, new controller.ChartTypeData(controller.ChartType.conversation)));
     tabElement.append(_conversationTabLink);
 
+    _driverTabLink = new ButtonElement()
```

Comment:
The CSS for showing which tab is active is incorrect when you click Drivers then another tab

Reply:
thanks for the catch! turns out I was clearing the `active` CSS class on the drivers tab. fixed now